ios/MullvadVPN/ApplicationConfiguration.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  ApplicationConfiguration.swift
+//  MullvadVPN
+//
+//  Created by pronebird on 05/06/2019.
+//  Copyright © 2019 Amagicom AB. All rights reserved.
+//
+
+import Foundation
+
+class ApplicationConfiguration {
```

Comment:
FYI Identifiers here are hardcoded and there is no any other way to access them at runtime.

- `securityGroupIdentifier` is used for accessing a shared container. The FileManager on iOS returns a container path for the given security group identifier. Both tunnel and the GUI app are in this security group so they both can access a shared container on the filesystem, but they need this identifier in our to fetch the right location on disk.

- `packetTunnelExtensionIdentifier` will be needed later to configure tunnels.